.golangci.yml

```diff
@@ -79,15 +79,18 @@ linters-settings:
     allow-multiline-assign: true
     allow-case-traling-whitespace: true
     allow-cuddle-declarations: false
-  godot:
-    check-all: false
 
 linters:
   enable-all: true
   disable:
     - testpackage
     - maligned
     - prealloc
+    - godot
```

Comment:
godot should be enabled

Reply:
https://github.com/hyperledger/aries-framework-go/pull/2181/commits/554a9286a8a0319cecbdf719569e74b09c14d383 

fixed in here